src/lib/components/progressbar/ProgressBar.component.js

```diff
@@ -0,0 +1,235 @@
+//@flow
+import React, { useEffect, useState } from 'react';
+import styled, { css } from "styled-components";
+
+export type ProgressBarProps = {
+  total: number,
+  filled: number,
+  size?: Size,
+  color: string,
+  measure: string,
+  percentage?: boolean,
+  capacity?: boolean,
+  label?: boolean
```

Comment:
- IMO, we do not need `total` and `filled` since we only need the `percentage` of the filled part. The percentage of the filled part should be computed outside and passed in props. We can use this percentage to set the `width` of the filled part.

- All the 4 labels should be passed in props too. They can not be hard-coded because we have to allow users to deal with Internationalization (i18n)

Suggested ProgressBarProps:

```
export type ProgressBarProps = {
  percentage: number, // 55
  size?: Size,
  color?: string, // default to primary color
  topLeftLabel?: string, // 55%
  topRightLabel?: string, // 80Gb Total
  bottomLeftLabel?: string, // 45Gb Used
  bottomRightLabel?: string, // 25Gb Free
}
```